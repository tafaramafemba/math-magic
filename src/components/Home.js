import { LoremIpsum } from 'react-lorem-ipsum';

function Home() {
  return (
    <div>
  <h1>Welcome to our Page</h1>
  <LoremIpsum p={2} />
    </div>
  );
}

export default Home;