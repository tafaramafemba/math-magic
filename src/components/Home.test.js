import renderer from 'react-test-renderer'
import Home from "./Home";

describe(' Home Component Rendering ', () =>{
    it('renders correctly', () => {
        const tree = renderer
            .create(<Home />)
        expect(tree).toMatchSnapshot();
    });
})