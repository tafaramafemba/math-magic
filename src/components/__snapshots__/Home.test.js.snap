// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Home Component Rendering  renders correctly 1`] = `
<div>
  <h1>
    Welcome to our Page
  </h1>
  <p>
    Lorem ipsum odor amet, consectetuer adipiscing elit. Odio fringilla hendrerit est auctor laoreet varius cubilia hendrerit. Semper iaculis tempus habitant potenti porta interdum at. Tempus porta tortor risus efficitur augue; libero hac diam. Urna habitant bibendum tortor dis est aenean. Facilisi vehicula ex mus risus himenaeos natoque aliquam. Cursus nascetur neque himenaeos molestie posuere lacus sagittis torquent.
  </p>
  <p>
    Dictum risus habitant phasellus purus turpis potenti sagittis luctus. Dis venenatis felis maximus nec netus efficitur etiam himenaeos. Lectus fringilla aenean fames justo mauris at est. Eros dolor fusce sapien; rhoncus velit varius. Consequat rhoncus imperdiet parturient aliquam mus orci quis augue. Porta sociosqu mollis egestas conubia donec sodales pretium felis. Dignissim risus rutrum hac tincidunt consequat. Neque convallis nascetur maecenas primis in. Rhoncus lectus sollicitudin sodales elit metus inceptos praesent nisl. Ante faucibus netus sollicitudin ornare integer feugiat risus parturient.
  </p>
</div>
`;
